ShowerShow/Control/DisclaimersController.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Microsoft.AspNetCore.Routing;
+using Microsoft.Azure.Functions.Worker.Http;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Attributes;
+using Microsoft.Extensions.Logging;
+using Microsoft.OpenApi.Models;
+using Newtonsoft.Json;
+using ShowerShow.Authorization;
+using ShowerShow.Controllers;
+using ShowerShow.Model;
+using ShowerShow.Models;
+using ShowerShow.Repository;
+using ShowerShow.Repository.Interface;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ShowerShow.Control
+{
+    public class DisclaimersController
+    {
```

Comment:
Why are disclaimers  not in extra function ?? It is in the wrong function app. Can you move these into extra function instead?